core/example/solvers/pme.cpp

```diff
@@ -0,0 +1,413 @@
+#include "definitions.h"
+#include "pme.h"
+#include <cmath>
+#include <gsl/gsl_sf_lambert.h>
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_Parallel.hpp>
+#include <Cabana_ExecutionPolicy.hpp>
+#include <atomic>
+#include <Kokkos_Pair.hpp>
+#include <fftw3.h>
+#include <complex>
+#include <vector>
+#include <time.h>
+#include <sys/time.h>
+
+using std::complex;
+using std::vector;
+
+
+TPME::TPME(double accuracy, ParticleList particles,ParticleList mesh, double lx, double ly, double lz)
```

Comment:
`mesh` does not get used in this constructor - does it get used elsewhere?

Reply:
Originally I figured `mesh` would be involved in the tuning, but that is not the case at the moment. I've removed `mesh` from the constructor now.